main.py

```diff
@@ -1,10 +1,98 @@
+import json
+from pathlib import Path
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
+BASE_DIR = Path(__file__).resolve().parent
+
+
+def create_guild(guild: dict):
+    if guild:
+        guild_name = guild["name"]
+        if not Guild.objects.filter(name=guild_name).exists():
+            if guild["description"]:
+                guild_description = guild["description"]
+                Guild.objects.create(
+                    name=guild_name,
+                    description=guild_description
+                )
+            else:
+                Guild.objects.create(name=guild_name)
+        return guild_name
```

Comment:
better return the `Guild` object here, not just `guild_name`. Then you can use this object and simplify your code